tmt.spec

```diff
@@ -30,6 +30,7 @@ This package contains the command line tool.
 %package -n     python%{python3_pkgversion}-%{name}
 Summary:        Python library for the %{summary}
 BuildRequires: python%{python3_pkgversion}-devel
+BuildRequires: python%{python3_pkgversion}-docutils
```

Comment:
The `rst2man` command is actually used during srpm creation:

https://github.com/teemtee/tmt/blob/05ae233cb5f30cf1532ec696fd3d044b933576b6/Makefile#L43

So it should not be needed during build. Or have I missed anything?

Reply:
Hm. tests/full uses `rlRun "dnf builddep -y tmt.spec" 0 "Install build dependencies"` and it doesn't install python3-docutils.
I'd would think that build deps would cover also "building srpm" use case

Reply:
Ok, I see, makes sense. And it will not hurt there.